ref-cache.md

```diff
@@ -2,3 +2,50 @@
 permalink: /ref-cache.html
 ---
 
+# Loc-I Reference Integrated Cache
```

Comment:
drop 'Integrated' from the title - its in the description

Reply:
ok done